src/js/lib/monster.ui.js

```diff
@@ -1465,6 +1466,28 @@ define(function(require) {
 			return form.valid();
 		},
 
+		/**
+		 * Convert the 'target' in to Markdown editor
+		 * @param target - mandatory jQuery object
+		 * @param [options] - optional object to overwrite the default configs
+		 */
```

Comment:
Our JSDoc method header provide at least both the [name and type](https://devdocs.io/jsdoc/tags-param#names-types-and-descriptions) of each parameter.

In this case, `target` is of type `jQuery` and `options` is of type `Object`.